src/helper.ts

```diff
@@ -165,6 +167,37 @@ async function getDeeplinkURL(): Promise<string | null> {
   }
 }
 
+export async function getPushToken(): Promise<PushTokenInfo | null> {
```

Comment:
Did not add a check for platform os before calling the ios specific functions, because they do not give an error when called for android, the promise safely resolves to null